Module not found with webpack 2 and blueprint 1.11.0

When using the new version of blueprint (1.11.0) with webpack 2 and I'm getting those errors : 
```
ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'React' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:27-43
 @ ./app/index.js

ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'window' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:45-62
 @ ./app/index.js

ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'ReactDOM' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:64-83
 @ ./app/index.js

ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'classNames' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:85-106
 @ ./app/index.js

ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'Tether' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:108-125
 @ ./app/index.js

ERROR in ./~/@blueprintjs/core/dist/core.bundle.js
Module not found: Error: Can't resolve 'React.addons.CSSTransitionGroup' in '/home/tibo/src/webpack-demo/node_modules/@blueprintjs/core/dist'
 @ ./~/@blueprintjs/core/dist/core.bundle.js 3:127-169
 @ ./app/index.js
```
I'm using webpack 2, 
here is my package.json : 
```
{
  "name": "webpack-demo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "webpack": "^2.2.1"
  },
  "dependencies": {
    "@blueprintjs/core": "^1.11.0",
    "lodash": "^4.17.4",
    "react": "^15.4.2",
    "react-addons-css-transition-group": "^15.4.2",
    "react-dom": "^15.4.2"
  }
}
```

and here is my `app/index.js`
```
import {Classes, Dialog, Tooltip} from "@blueprintjs/core";
import * as Blueprint from "@blueprintjs/core";
function component () {
          var element = document.createElement('div');

          /* lodash is required for the next line to work */
          element.innerHTML = _.join(['Hello','webpack'], ' ');

          return element;
}

document.body.appendChild(component());
```
and my `index.html` : 
```
<html>
        <head>
                <title>webpack 2 demo</title>
                <script src="https://unpkg.com/lodash@4.16.6"></script>
        </head>
        <body>
                <script src="dist/bundle.js"></script>
        </body>
</html>
```

and I'm running : 
```
webpack app/index.js dist/bundle.js
```

Everything is working fine with blueprint 1.10.0...

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -5,8 +5,7 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint.css",
-  "browser": "dist/core.bundle.js",
-  "webpack": "dist/index.js",
+  "unpkg": "dist/core.bundle.js",
   "dependencies": {
     "@types/dom4": "^1.5.20",
     "@types/tether": "^1.1.27",

--- a/packages/datetime/package.json
+++ b/packages/datetime/package.json
@@ -5,8 +5,7 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint-datetime.css",
-  "browser": "dist/datetime.bundle.js",
-  "webpack": "dist/index.js",
+  "unpkg": "dist/datetime.bundle.js",
   "dependencies": {
     "@blueprintjs/core": "^1.3.0",
     "classnames": "^2.2",

--- a/packages/table/package.json
+++ b/packages/table/package.json
@@ -5,8 +5,7 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/table.css",
-  "browser": "dist/table.bundle.js",
-  "webpack": "dist/index.js",
+  "unpkg": "dist/table.bundle.js",
   "scripts": {
     "build:gulp": "$(cd ..; npm bin)/gulp tsc-table sass-table",
     "build:preview": "webpack --config config/webpack.config.preview.js",

